OLE/ChainedBlockStream.php

```diff
@@ -160,7 +160,7 @@ function stream_read($count)
         if ($this->stream_eof()) {
             return false;
         }
-        $s = substr($this->data, $this->pos, $count);
+        $s = substr($this->data, $this->pos ?? 0, $count);
```

Comment:
I'm afraid this is not going to fly on PHP 5.6 which we still support :cry:

Reply:
Oh! I try fixed it.